Example/IGListKitExamples/ViewControllers/MixedDataViewController.swift

```diff
@@ -71,13 +71,8 @@ class MixedDataViewController: UIViewController, IGListAdapterDataSource {
     //MARK: IGListAdapterDataSource
 
     func objects(for listAdapter: IGListAdapter) -> [IGListDiffable] {
-        var items = [IGListDiffable]()
-        for obj in data {
-            if selectedClass == nil || type(of: obj) == selectedClass! {
-                items.append(obj as! IGListDiffable)
-            }
-        }
-        return items
+        guard selectedClass != nil else { return data.map { $0 as! IGListDiffable } }
```

Comment:
nit: I think this would be easier to read if we put the `return` on a newline and the final `}` on a newline